SampleApp.Common/BaseStartup.cs

```diff
@@ -42,6 +42,10 @@ public virtual void Configure(IApplicationBuilder app)
             app.Use((context, next) =>
             {
                 context.Request.HttpContext.Connection.RemoteIpAddress ??= IPAddress.Parse("192.168.0.1");
+                if (context.Request.Headers.ContainsKey("X-Forwarded-For"))
```

Comment:
this is diff from what the docs show? should we update those docs that Hans added?

Reply:
This is because my e2e tests mock actual http requests, resulting in my request not having an ip, so in the sample apps that are used in the e2e tests, I manually setup the ip here. In actual request, you wouldn't do that,  comment.
```suggestion
                // we manually do some Ip setup here, because our e2e tests mock http requests and have some missing request info becuase of that (no ip address for example)
                // for proper forwarding setup in real environments, see https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/proxy-load-balancer?view=aspnetcore-9.0&preserve-view=true
                if (context.Request.Headers.ContainsKey("X-Forwarded-For"))
```